app/src/main/res/layout-land/activity_onboarding.xml

```diff
@@ -121,6 +121,18 @@
         app:layout_constraintTop_toBottomOf="@+id/onboarding_viewpager"
         />
 
```

Comment:
In smaller devices ( nexus S) the skip button overlaps the recycler view. Consider adding a margin between views